restructuredtext_lint/cli.py

```diff
@@ -5,20 +5,36 @@
 import os
 import sys
 
+from collections import OrderedDict
```

Comment:
`collections` is built-in so it can be relocated along-side `argparse`, `json`, etc (i.e. in the first party packages)